2022-12-23 03:54:03,826 INFO    MainThread:16038 [wandb_setup.py:_flush():68] Configure stats pid to 16038
2022-12-23 03:54:03,826 INFO    MainThread:16038 [wandb_setup.py:_flush():68] Loading settings from /home/codespace/.config/wandb/settings
2022-12-23 03:54:03,826 INFO    MainThread:16038 [wandb_setup.py:_flush():68] Loading settings from /workspaces/codespaces-jupyter/notebooks/wandb/settings
2022-12-23 03:54:03,826 INFO    MainThread:16038 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True', 'api_key': '***REDACTED***'}
2022-12-23 03:54:03,826 INFO    MainThread:16038 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-12-23 03:54:03,827 INFO    MainThread:16038 [wandb_init.py:_log_setup():476] Logging user logs to /workspaces/codespaces-jupyter/notebooks/wandb/run-20221223_035403-1ba97j35/logs/debug.log
2022-12-23 03:54:03,827 INFO    MainThread:16038 [wandb_init.py:_log_setup():477] Logging internal logs to /workspaces/codespaces-jupyter/notebooks/wandb/run-20221223_035403-1ba97j35/logs/debug-internal.log
2022-12-23 03:54:03,827 INFO    MainThread:16038 [wandb_init.py:init():516] calling init triggers
2022-12-23 03:54:03,827 INFO    MainThread:16038 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {'use_amp': False, 'epochs': 10, 'batch_size': 3, 'lr': 0.001, 'momentum': 0.9, 'dataset': 'CIFAR10', 'architecture': 'CNN'}
2022-12-23 03:54:03,827 INFO    MainThread:16038 [wandb_init.py:init():535] re-initializing run, found existing run on stack: ghpgpsjn
2022-12-23 03:54:03,829 INFO    MainThread:16038 [wandb_run.py:_finish():1753] finishing run ecust-bestcv/image-classifier/ghpgpsjn
2022-12-23 03:54:03,829 INFO    MainThread:16038 [jupyter.py:save_history():448] not saving jupyter history
2022-12-23 03:54:03,830 INFO    MainThread:16038 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-12-23 03:54:03,830 INFO    MainThread:16038 [wandb_init.py:_jupyter_teardown():408] cleaning up jupyter logic
2022-12-23 03:54:03,830 INFO    MainThread:16038 [wandb_run.py:_atexit_cleanup():1955] got exitcode: 0
2022-12-23 03:54:03,830 INFO    MainThread:16038 [wandb_run.py:_restore():1938] restore
2022-12-23 03:54:03,830 INFO    MainThread:16038 [wandb_run.py:_restore():1944] restore done
2022-12-23 03:54:09,857 INFO    MainThread:16038 [wandb_run.py:_footer_history_summary_info():3408] rendering history
2022-12-23 03:54:09,858 INFO    MainThread:16038 [wandb_run.py:_footer_history_summary_info():3440] rendering summary
2022-12-23 03:54:09,862 INFO    MainThread:16038 [wandb_run.py:_footer_sync_info():3364] logging synced files
2022-12-23 03:54:09,867 INFO    MainThread:16038 [wandb_init.py:init():569] starting backend
2022-12-23 03:54:09,868 INFO    MainThread:16038 [wandb_init.py:init():573] setting up manager
2022-12-23 03:54:09,871 INFO    MainThread:16038 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-12-23 03:54:09,873 INFO    MainThread:16038 [wandb_init.py:init():580] backend started and connected
2022-12-23 03:54:09,891 INFO    MainThread:16038 [wandb_run.py:_label_probe_notebook():1116] probe notebook
2022-12-23 03:54:09,891 INFO    MainThread:16038 [wandb_run.py:_label_probe_notebook():1126] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-12-23 03:54:09,891 INFO    MainThread:16038 [wandb_init.py:init():658] updated telemetry
2022-12-23 03:54:09,897 INFO    MainThread:16038 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2022-12-23 03:54:11,414 INFO    MainThread:16038 [wandb_run.py:_on_init():2006] communicating current version
2022-12-23 03:54:11,458 INFO    MainThread:16038 [wandb_run.py:_on_init():2010] got version response 
2022-12-23 03:54:11,458 INFO    MainThread:16038 [wandb_init.py:init():728] starting run threads in backend
2022-12-23 03:54:11,491 INFO    MainThread:16038 [wandb_run.py:_console_start():1986] atexit reg
2022-12-23 03:54:11,491 INFO    MainThread:16038 [wandb_run.py:_redirect():1844] redirect: SettingsConsole.WRAP_RAW
2022-12-23 03:54:11,492 INFO    MainThread:16038 [wandb_run.py:_redirect():1909] Wrapping output streams.
2022-12-23 03:54:11,492 INFO    MainThread:16038 [wandb_run.py:_redirect():1931] Redirects installed.
2022-12-23 03:54:11,493 INFO    MainThread:16038 [wandb_init.py:init():765] run started, returning control to user process
